components/button/button.css

```diff
@@ -0,0 +1,178 @@
+@import url(./shared/reset.css);
+@import url(./shared/style-guide.css);
+@import url(./shared/utility.css);
+@import url(./shared/iframe.css);
+
+/* solid buttons  */
+
+.button {
+  width: fit-content;
+  padding: calc(var(--spc-s) - 0.3rem);
+  border-radius: var(--br-s);
+  border: none;
+  color: white;
+  text-align: center;
+  cursor: pointer;
+  position: relative;
+}
+
+.button--primary {
+  background-color: var(--primary);
+}
+
+.button--secondary {
+  background-color: var(--secondary);
+}
+
+.button--danger {
+  background-color: var(--danger);
+}
+
+.button--success {
+  background-color: var(--success);
+}
+
+.button--warning {
+  background-color: var(--warning);
+}
+
+.button--info {
+  background-color: var(--info);
+}
+
+.button--dark {
+  background-color: var(--dark);
+}
+
+.button--light {
+  background-color: var(--light-color);
+  color: var(--dark);
+}
+.no-br {
+  border-radius: 0px;
+}
+
+/* links  */
+
+.primary-link {
+  color: var(--primary);
+}
+.secondary-link {
+  color: var(--secondary);
+}
+/* outline buttons  */
+.outline-button-primary {
+  color: var(--primary);
+  background-color: white;
+  border: 1px solid currentColor;
+}
+
+.outline-button-secondary {
+  color: var(--secondary);
+  background-color: white;
+  border: 1px solid currentColor;
+}
+
+.outline-button-danger {
+  color: var(--danger);
+  background-color: white;
+  border: 1px solid currentColor;
+}
+
+.outline-button-success {
+  color: var(--success);
+  background-color: white;
+  border: 1px solid currentColor;
+}
+
+.outline-button-warning {
+  color: var(--warning);
+  background-color: white;
+  border: 1px solid currentColor;
+}
```

Comment:
You can use a common class for background-color & border.

Reply:
sure. Thanks.